src/index.js

```diff
@@ -0,0 +1,71 @@
+import { getNewProjects, getProjectsWithComments, getUpcomingProjects } from './queryService.js';
+import { format } from 'date-fns';
+import mailClient from '@sendgrid/mail';
+
+if (!['production', 'test'].includes(process.env.NODE_ENV)) {
+  await import('dotenv/config');
+}
+
+if (process.env.NODE_ENV !== 'test') {
+  const fs = await import('fs');
+  let apiKey = undefined;
+  try {
+    apiKey = fs.readFileSync('/secrets/email/key', 'utf8');
+    console.log('found in root');
+  } catch {
+    console.warn('secret not found at root. Checking local');
+  }
+
+  try {
+    apiKey = fs.readFileSync('secrets/email/key', 'utf8');
```

Comment:
**nitpick**: Do you want to add an if statement so that it's not needlessly trying to read this file if it was already found at the root?

Reply:
I think if it found both, i'd want the last one to take precedence because the root one could be completely wrong. It only runs once a week so I don't expect any performance issues.